docs/configuration/configuration.md

```diff
@@ -1359,7 +1359,7 @@ interface.
 The following meta labels are available on targets during [relabeling](#relabel_config):
 
 * `__meta_openstack_address_pool`: the pool of the private IP.
-* `__meta_openstack_instance_flavor`: the flavor ID of the OpenStack instance.
+* `__meta_openstack_instance_flavor`: the flavor Name or flavor ID of the OpenStack instance (flavor Name takes precedence over ID).
```

Comment:
I don't think we can talk about precedence in here as the two attributes are mutually exclusive.

```suggestion
* `__meta_openstack_instance_flavor`: the flavor name or flavor ID of the OpenStack instance, whichever is available
```